extensions/applicationinsights-properties-js/Context/Internal.ts

```diff
@@ -18,6 +19,6 @@ export class Internal implements IInternal {
     * Constructs a new instance of the internal telemetry data class.
     */
     constructor(config: ITelemetryConfig) {
-        this.sdkVersion = (config.sdkExtension() ? config.sdkExtension() + "_" : "") + "javascript:" + ApplicationInsights.Version;
+        this.sdkVersion = (config.sdkExtension && config.sdkExtension() ? config.sdkExtension() + "_" : "") + "javascript:" + Version;
```

Comment:
is this where we are getting the a_ in the telemetry for existing SDK today?

Reply:
I don't remember seeing an 'a_' in tags for current SDK. Here's a sample from the current SDK:
ai.internal.sdkVersion=javascript:1.0.17

---
In reply to: [214087493](https://github.com/Microsoft/ApplicationInsights-JS/pull/660#discussion_r214087493) [](ancestors = 214087493)